CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/MainActivity.java

```diff
@@ -56,10 +56,12 @@ public static String getLocalIpAddress() {
         try {
             for (Enumeration<NetworkInterface> en = NetworkInterface.getNetworkInterfaces(); en.hasMoreElements();) {
                 NetworkInterface intf = en.nextElement();
-                System.out.println(intf.getName());
+                String intf_name = intf.getName();
+                System.out.println(intf_name);
                 for (Enumeration<InetAddress> enumIpAddr = intf.getInetAddresses(); enumIpAddr.hasMoreElements();) {
                     InetAddress inetAddress = enumIpAddr.nextElement();
-                    if (!inetAddress.isLoopbackAddress() && intf.getName().equals("eth1") && (inetAddress instanceof Inet4Address)) {
+                    String name = intf_name;
```

Comment:
intf_name is declared and set value outside of the "inner" for loop, its value is not changed inside the "inner" for loop. assigning it to another variable "String name" seems not necessary and variable "name" is not used. Shall the line
String name = intf_name 
get removed?

Reply:
removed.